examples/bingo/base-image/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM vmware/dispatch-openfaas-nodejs6-base:0.0.3-dev1
+
+RUN npm install request@2.83.0
```

Comment:
I actually need similar image for https://github.com/vmware/dispatch/blob/master/examples/nodejs6/slack_vm_being_deployed.js 
Should we create few more "common" images in `vmware` org (or some separate org, like dispatch) until we add the feature to build images based on requirements.txt to dispatch? CC @berndtj